src/tts/magpietts/model.cpp

```
@@ -2114,32 +2189,55 @@ MagpieCodeGenerator::generate(
-            const double inter_ms = metrics.record_frame(frame_done_us, first_frame);
-            const double frame_latency_ms = (double)(frame_done_us - frame_start_us) / 1000.0;
-            if (step < 4 || step % 10 == 0) {
-                if (first_frame) {
-                    fprintf(
-                        stderr, "%s frame %d latency=%.2f ms ttff=%.2f ms\n", label, step,
-                        frame_latency_ms, metrics.ttff_ms);
-                } else {
-                    fprintf(
-                        stderr, "%s frame %d latency=%.2f ms inter=%.2f ms\n", label, step,
-                        frame_latency_ms, inter_ms);
+            if (frames_to_emit > 0) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)